index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Javascript Society Summit - Conference Web Page</title>
+</head>
+
+<body>
+    <!--HEADER-->
+    <div class="background">
+        <header class="header">
+            <div class="dark-nav">
+                <ul>
+                    <li><a href="https://www.facebook.com/"><img src="images/facebook.png" alt="fcebook icon"></a></li>
+                    <li><a href="https://twitter.com/home"><img src="images/twitter.png" alt="twitter icon"></a></li>
+                    <li><a href="/">English</a></li>
+                    <li><a href="/">My Page</a></li>
+                    <li><a href="/">Log out</a></li>
+                </ul>
+            </div>
+
+            <nav class="navbar">
+                <a href="index.html" class="nav-branding"><img src="images/dark logo.png" alt="logo"></a>
+
+                <ul class="nav-menu">
+                    <li class="nav-item">
+                        <a href="about.html" class="nav-link orange-text">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#program" class="nav-link">Program</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#featured" class="nav-link">Featured</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#sponsor" class="nav-link">Sponsor</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="/" class="nav-link border-text">Community</a>
+                    </li>
```

Comment:
As stated by the previous reviewer, you should add margin or padding to the right side of the second navigation or reduce that of the first navigation. as the two navigations are supposed to be on the same line vertically as it is in the design given

Design given:
![image](https://github.com/peter-mwovi/Capstone-project/assets/20537712/ba961267-4ed5-4700-a2d9-6c3fe37d229b)

Your design :
![image](https://github.com/peter-mwovi/Capstone-project/assets/20537712/a0251474-9bf0-4c9f-ac3f-152aa141cdbd)